File: include/v1_settings.cs
using System;
using System.IO;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Web;
using System.Web.Mvc;
using System.Reflection;
using runnerDotNet;
namespace runnerDotNet
{
	public static partial class Settings_v1
	{
		static public void Apply()
		{
			SettingsMap fieldLabelsArray = new XVar(), pageTitlesArray = new XVar(), query = null, queryData_Array = new XVar(), table = null, tdataArray = new XVar();
			tdataArray["v1"] = SettingsMap.GetArray();
			tdataArray["v1"][".ShortName"] = "v1";
			tdataArray["v1"][".pagesByType"] = MVCFunctions.my_json_decode(new XVar("{\"dashboard\":[\"dashboard\"]}"));
			tdataArray["v1"][".originalPagesByType"] = tdataArray["v1"][".pagesByType"];
			tdataArray["v1"][".pages"] = CommonFunctions.types2pages(MVCFunctions.my_json_decode(new XVar("{\"dashboard\":[\"dashboard\"]}")));
			tdataArray["v1"][".originalPages"] = tdataArray["v1"][".pages"];
			tdataArray["v1"][".defaultPages"] = MVCFunctions.my_json_decode(new XVar("{\"dashboard\":\"dashboard\"}"));
			tdataArray["v1"][".originalDefaultPages"] = tdataArray["v1"][".defaultPages"];
			fieldLabelsArray["v1"] = SettingsMap.GetArray();
			pageTitlesArray["v1"] = SettingsMap.GetArray();
			if(CommonFunctions.mlang_getcurrentlang() == "English")
			{
				fieldLabelsArray["v1"]["English"] = SettingsMap.GetArray();
			}
			tdataArray["v1"][".shortTableName"] = "v1";
			tdataArray["v1"][".entityType"] = 4;


			tdataArray["v1"][".hasEvents"] = true;
			tdataArray["v1"][".tableType"] = "dashboard";

			tdataArray["v1"][".addPageEvents"] = false;
			tdataArray["v1"][".isUseAjaxSuggest"] = true;
			GlobalVars.tables_data["v1"] = tdataArray["v1"];
			GlobalVars.field_labels["v1"] = fieldLabelsArray["v1"];
			GlobalVars.page_titles["v1"] = pageTitlesArray["v1"];
			GlobalVars.tables_data["v1"] = tdataArray["v1"];
		}
	}

}

File: include/v1_settings.php
<?php
require_once(getabspath("classes/cipherer.php"));



$tdatav1 = array();
$tdatav1[".ShortName"] = "v1";

$tdatav1[".pagesByType"] = my_json_decode( "{\"dashboard\":[\"dashboard\"]}" );
$tdatav1[".originalPagesByType"] = $tdatav1[".pagesByType"];
$tdatav1[".pages"] = types2pages( my_json_decode( "{\"dashboard\":[\"dashboard\"]}" ) );
$tdatav1[".originalPages"] = $tdatav1[".pages"];
$tdatav1[".defaultPages"] = my_json_decode( "{\"dashboard\":\"dashboard\"}" );
$tdatav1[".originalDefaultPages"] = $tdatav1[".defaultPages"];


//	field labels
$fieldLabelsv1 = array();
$pageTitlesv1 = array();

if(mlang_getcurrentlang()=="English")
{
	$fieldLabelsv1["English"] = array();
}

/*
//	search fields
$tdatav1[".searchFields"] = array();

// all search fields
$tdatav1[".allSearchFields"] = array();

// good like search fields
$tdatav1[".googleLikeFields"] = array();
*/

/*
$tdatav1[".dashElements"] = array();

*/
$tdatav1[".shortTableName"] = "v1";
$tdatav1[".entityType"] = 4;





include_once(getabspath("include/v1_events.php"));
$tdatav1[".hasEvents"] = true;


$tdatav1[".tableType"] = "dashboard";



$tdatav1[".addPageEvents"] = false;

$tdatav1[".isUseAjaxSuggest"] = true;

$tables_data["v1"]=&$tdatav1;
$field_labels["v1"] = &$fieldLabelsv1;
$page_titles["v1"] = &$pageTitlesv1;

?>